recipes/ofeli/all/conanfile.py

```diff
@@ -0,0 +1,75 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+import os
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.40.0"
+
+
+class OfeliConan(ConanFile):
+    name = "ofeli"
+    description = "An Object Finite Element Library"
+    topics = ("ofeli", "finite element", "finite element library",
+              "finite element analysis", "finite element solver")
+    license = "LGPL-3.0-or-later"
+    homepage = "http://ofeli.org/index.html"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _doc_folder(self):
+        return os.path.join(
+            self._source_subfolder,
+            "doc"
+        )
+
+    def validate(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration(
+                "Ofeli is just supported for Linux")
+        if self.settings.compiler != "gcc":
+            raise ConanInvalidConfiguration(
+                "Ofeli is just supported for GCC")
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 11)
+        if self.settings.compiler.libcxx != "libstdc++11":
```

Comment:
Which scenario it doesn't work? I'm able to with gcc and libstdc++ without any error.

Reply:
cf https://github.com/conan-io/conan-center-index/pull/8375#discussion_r777419520
The failure is the one happening in https://c3i.jfrog.io/c3i/misc/summary.html?json=https://c3i.jfrog.io/c3i/misc/logs/pr/8375/8-configs/linux-gcc/ofeli/4.1.2//summary.json for example
This is expected, because the upstream project ovewrites CPPFLAGS passed by conan, so it does not respect the profile regarding the cpp standard used, or libstdc++'s abi, so it ends up using the deafult, which is libstdc++11

Reply:
Thank you for these details!